backend/src/services/user.service.ts

```diff
@@ -1,13 +1,19 @@
-import User, {UserModel} from '../models/user.model';
+import User, { UserModel } from "../models/user.model";
 
 export const createUser = async (userDetails: UserModel) => {
-    const user = new User(userDetails);
-    await user.save();
-    return user;
+  const user = new User(userDetails);
+  await user.save();
+  return user;
+};
+
+export const getUserByAuthId = async (uid) => {
+  const user = await User.findOne({ auth_id: uid });
+  return user;
```

Comment:
What happens when we dont find a user with that auth_id? What gets returned?

Reply:
`null` would get returned and there would have to be a `if (user != null)` check inside whichever controller requests the user I think